libheif/heif_encoder_x265.cc

```diff
@@ -633,7 +633,14 @@ static struct heif_error x265_encode_image(void* encoder_raw, const struct heif_
 #endif
 
   int s = snprintf(ctu_buf,4,"%d",ctuSize);
-  assert(s<4);
```

Comment:
Should probably be `s >= 4`.
I also think that instead of testing the output of `snprintf` here, `ctuSize` should be tested directly to be a power of two in the range [16;64].

Reply:
As -DNDEBUG is set, the assert is not compiled in if you build a release version (-O2) and you would still get the unused variable error ...

Reply:
Ah, sorry. You mean the if check, yes you're right. Fixed.